app.js

```diff
@@ -1,15 +1,20 @@
 const express = require("express");
-const { getTopics } = require("./controllers/app.controller");
+const { getTopics, getArticles, getArticleById } = require("./controllers/app.controller");
 
 const app = express();
 
 app.use(express.json());
 
```

Comment:
app.use(express.json()) is only useful to us when we have sent data from testing(or client side) to the app. I _wonder_ why you have put this in early ... 🤔 📸